src/main/java/org/folio/fqm/repository/IdStreamer.java

```diff
@@ -63,12 +71,12 @@ public int streamIdsInBatch(UUID queryId,
                               Consumer<IdsWithCancelCallback> idsConsumer) {
     UUID entityTypeId = queryDetailsRepository.getEntityTypeId(queryId);
     EntityType entityType = getEntityType(entityTypeId);
-    Condition condition = field(ID_FIELD_NAME).in(
+    Condition condition = field("\"source1\".id").in(
```

Comment:
This seems suspicious...

Reply:
After some digging, it looks like this method is only actually used for tests, so it's secretly dead code in disguise. I'll go ahead and remove it.

Reply:
Yep, I think I changed this at some point just to get things to stop complaining, then forgot to ever fix it since nothing actually uses it 😆